src/tools/perf/perftest.c

```diff
@@ -449,6 +455,12 @@ static ucs_status_t setup_sock_rte_p2p(struct perftest_context *ctx)
             status = UCS_ERR_IO_ERROR;
             goto err_close_connfd;
         }
+
+        if (ctx->params.super.api == UCX_PERF_API_UCT) {
```

Comment:
do we need to do such backup for the three supported RTE (mad/sock/mpi)?

Reply:
I don't see if mpi does exchange of parameters, but it might be relevant for mad